modules/indexing/src/main/java/org/apache/ignite/internal/mxbean/SqlQueryMXBean.java

```diff
@@ -178,6 +178,14 @@
         "heavy analytics query.")
     String getSqlQueryMemoryQuota();
 
+    /**
+     * Gets the amount of memory available for sql queries.
+     *
+     * @return sql query available memory as a long.
+     */
+    @MXBeanDescription("How much memory in bytes currently left available for the queries on this node.")
+    Long getSqlFreeMem();
```

Comment:
Please don't shorten public API names.

I suggest we call this akin to the quotas methods
```suggestion
    Long getSqlFreeMemoryBytes();
```